ArduCopter/ArduCopter.cpp

```diff
@@ -120,7 +124,9 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if FRAME_CONFIG == HELI_FRAME
     SCHED_TASK(check_dynamic_flight,  50,     75),
 #endif
+#if LOGGING_ENABLED == ENABLED
     SCHED_TASK(fourhundred_hz_logging,400,    50),
```

Comment:
Maybe move this down to the other logging calls?

Reply:
I avoided do that functional change, but if you say it is OK, sure, I can do that.